tests/source/issue-4452.rs

```diff
@@ -0,0 +1,15 @@
+fn a(&self) -> i64 {
+    #[allow(clippy::cast_possible_wrap)]
+    1u64 as i64
+}
+
+fn b(&self) -> i64 {
+    #[allow(clippy::cast_possible_wrap)]
+    #[attr]
+    1..2
+}
+
+fn foo() {
+    #[allow(clippy::cast_possible_wrap)]
+    {}
+}
```

Comment:
Can remove this file and just utilize the target since the purpose of this test is validating span correctness and the maintenance of comments